include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -893,4 +928,115 @@ operator()(
                 });
     }
 }
+
+template <
+        class ExecSpace,
+        class MemorySpace,
+        class BSplines,
+        class InterpolationDDim,
+        ddc::BoundCond BcLower,
+        ddc::BoundCond BcUpper,
+        SplineSolver Solver,
+        class... IDimX>
+std::tuple<
+        ddc::Chunk<
+                double,
+                ddc::DiscreteDomain<
+                        ddc::Deriv<typename InterpolationDDim::continuous_dimension_type>>,
+                ddc::KokkosAllocator<double, MemorySpace>>,
+        ddc::Chunk<
+                double,
+                ddc::DiscreteDomain<InterpolationDDim>,
+                ddc::KokkosAllocator<double, MemorySpace>>,
+        ddc::Chunk<
+                double,
+                ddc::DiscreteDomain<
+                        ddc::Deriv<typename InterpolationDDim::continuous_dimension_type>>,
+                ddc::KokkosAllocator<double, MemorySpace>>>
+SplineBuilder<
+        ExecSpace,
+        MemorySpace,
+        BSplines,
+        InterpolationDDim,
+        BcLower,
+        BcUpper,
+        Solver,
+        IDimX...>::quadrature_coefficients() const
+{
+    // Compute integrals of bsplines
+    ddc::Chunk<double, ddc::DiscreteDomain<bsplines_type>> integral_bsplines(spline_domain());
+    ddc::discrete_space<bsplines_type>().integrals(integral_bsplines.span_view());
```

Comment:
Why is this calculated on CPU even if the ExecSpace/MemorySpace of this class is GPU?

Reply:
Everything is computed on CPU (because we work on a single thread (single rhs) then copied on GPU here:

https://github.com/CExA-project/ddc/pull/560/files#diff-a120a1119000f5d36cb8a669f09e5b36ac36fdea28981ae6ae5f034f60e3801dR1016

(I said the contrary in the Gysela MR but this was a mistake).